geode-core/src/main/java/org/apache/geode/internal/cache/execute/FunctionRemoteContext.java

```diff
@@ -86,7 +86,12 @@ public void fromData(DataInput in) throws IOException, ClassNotFoundException {
     }
     this.isReExecute = DataSerializer.readBoolean(in);
 
-    if (StaticSerialization.getVersionForDataStream(in).isNotOlderThan(KnownVersion.GEODE_1_14_0)) {
+    KnownVersion dataStreamVersion = StaticSerialization.getVersionForDataStream(in);
+    if (dataStreamVersion.isNewerThanOrEqualTo(KnownVersion.GEODE_1_14_0)
```

Comment:
Do you think it would be more readable to define a new method like.
```
dataStreamVersion.isBetween(KnownVersion.GEODE_1_12_1, KnownVersion.GEODE_1_13_0)
```
My concern with such a method is questioning which terms are inclusive or exclusive. The above is `[KnownVersion.GEODE_1_12_1, KnownVersion.GEODE_1_13_0)`. Given that the eldest in the 1.12.x isn't know we can't define a constantly inclusive range either. We can define a consistently exclusive range, `(KnownVersion.GEODE_1_12_0, KnownVersion.GEODE_1_13_0)`.

```
dataStreamVersion.isBetweenExclusive(KnownVersion.GEODE_1_12_0, KnownVersion.GEODE_1_13_0)
```

Reply:
Yeah - if it starts happening in more places we can add something more saner.